save-backend/src/main/kotlin/com/saveourtool/save/backend/service/vulnerability/VulnerabilityService.kt

```diff
@@ -82,6 +82,12 @@ class VulnerabilityService(
         return vulnerabilityMetadataRepository.countByUserIdAndStatus(user.requiredId(), status)
     }
 
+    /**
+     * @param status status of vulnerabilities that should be counted
+     * @return count of vulnerabilities
+     */
+    fun countByStatus(status: VulnerabilityStatus): Int = vulnerabilityMetadataRepository.countByStatus(status)
```

Comment:
You can use the ```getCountVulnerabilities``` method. It is enough to pass ```VulnerabilityFilter.empty``` to it.

Reply:
yes